payjoin-directory/src/error.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Debug)]
+pub enum DbPoolError {
```

Comment:
I think this needs to be `pub(crate)` since we don't plan to expose it in the public API.